### DISCUSSION TITLE
The next new plugin

### DISCUSSION THREAD

**Discussion by d4x1** [apache/devlake#6619] [Polls]
Althrough DevLake supports many plugins https://devlake.apache.org/docs/Plugins , there are still many tools to be supported.
Here is a vote that can help DevLake team to decide which is the next one to be supported.

Feel free to comment your opinions.

**Comment by Startrekzky** [comment]
@d4x1 This vote looks great. As far as I know, SonarCloud, Trello, and ServiceNow are the most wanted plugins based on the feedback from the community. You can add it to the options.

I think all of them already have the corresponding issues. You can search for the issue number if you want.

**Reply by d4x1** [reply]
It's a pity that max options count is 8, there are so many plugins to be added.

**Comment by d4x1** [comment]
https://github.com/apache/incubator-devlake/issues?q=is%3Aopen+is%3Aissue+label%3Aadd-a-plugin

**Comment by surevs** [comment]
can you update this for multiple options to be selected - Tekton, ArgoCD and ServiceNow

**Reply by d4x1** [reply]
I don't find any configuration that can make it multiple.

**Comment by mlagergren** [comment]
Linear.app :-)

**Comment by d4x1** [comment]
kanbanize (from #7616 )

**Comment by kzhou57** [comment]
kanbanize +1

**Comment by adamlyka** [comment]
linear.app !!!

**Comment by AnkushVishwakarma01** [comment]
I want to add ServiceNow plugin. can someone help.

**Reply by d4x1** [reply]
@AnkushVishwakarma01  Welcome to contribute code, let me know if I can help.
